src/main/resources/dsl/workflow1.yaml

```diff
@@ -0,0 +1,27 @@
+# This sample workflow execute 3 steps in sequence.
+# 1) sampleActivity1, takes arg1 as input, and put result as result1.
+# 2) sampleActivity2, takes result1 as input, and put result as result2.
+# 3) sampleActivity3, takes args2 and result2 as input, and put result as result3.
+
+variables:
+  arg1: value1
+  arg2: value2
+
+root:
+  sequence:
+    elements:
+      - activity:
+          name: GetInfo1
```

Comment:
Shouldn't this be the ActivityImpl's Name, vs the activity entry point method name

Reply:
It has to be activity type which is in Java is defined (by default) as a method name (with the first letter capitalized) of an interface annotated with @ActivityInterface.